### PR TITLE
my.nextdns.io log images fix

### PR DESCRIPTION
**Pull request by FadeMind**
Site: my.nextdns.io/ `<id-config>` /logs

Fix
Domains

Generic

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7559,6 +7559,7 @@ img[src*="/static/media/sonos"]
 .settings-button
 .stream-button
 .tooltipParent > img
+.list-group-item > div > img
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7562]: Why invert logo's I think we actually should `IGNORE IMAGE ANALYSIS` logo's.

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:7562]: ¯\\\_(ツ)\_/¯

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7562]: Well your inverting an inverted images making it practically none inverted but what happens when it's not inverted anymore? then we make it inverted.

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:7562]: Without selector `.list-group-item > div > img` images in logs are normal - **not inverted**.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7562]: Can I ask then why we are inverting it?

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:7562]: Dark icons used by some domains are in default black. Not easly recognize. That's why invert is good IMHO.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7562]: Okay.

**Review by Gusted — APPROVED** [review]

**Pull request merged**